src/app/components/rubric-action-button-group/rubric-action-button-group.component.spec.ts

```diff
@@ -33,9 +33,45 @@ describe('RubricActionButtonGroupComponent', () => {
         fixture.detectChanges();
     });
 
-    it('should render buttons', () => {
+    it('should render 2 buttons when its parent is list-item', () => {
+        component.floatRight = true;
+
+        const buttons = fixture.nativeElement.querySelectorAll('button');
+
+        void expect(buttons.length).toEqual(2);
+    });
+
+    it('should render 3 buttons when its parent is rubric-header', () => {
+        component.showPrint = true;
+        fixture.detectChanges();
+
         const buttons = fixture.nativeElement.querySelectorAll('button');
-        void expect(buttons.length).toBeGreaterThanOrEqual(2);
-        void expect(buttons.length).toBeLessThanOrEqual(4);
+
+        void expect(buttons.length).toEqual(3);
+    });
+
+    it('should set copied to true when copyLink is called', () => {
+        component.copyLink();
+
+        void expect(component.copied).toEqual(true);
```

Comment:
I would add a spy and assertion that `document.execCommand('copy')` was called as well.

There are times when a single function does multiple things, so while I don't think it's necessary to write expectations for _every_ side effect, the `copy` command is essentially the key part of this function. It tells the user's browser to copy the link.